Dmf/Modules.Library/Dmf_ConnectedStandby.md

```diff
@@ -0,0 +1,112 @@
+## DMF_ConnectedStandby
+
+-----------------------------------------------------------------------------------------------------------------------------------
+
+#### Module Summary
+
+-----------------------------------------------------------------------------------------------------------------------------------
+
+This Module allows the Client to request and receive notifications when the system has entered or exited Connected Standby.
```

Comment:
Windows 10 Modern Standby (Modern Standby) expands the Windows 8.1 Connected Standby power model

Reply:
fixed